great_expectations/expectations/expectation.py

```diff
@@ -1752,6 +1752,47 @@ def _isclose(
         return np.isclose(operand_a_as_number, operand_b_as_number)
 
 
+class QueryExpectation(TableExpectation, ABC):
+
+    default_kwarg_values = {
+        "result_format": "BASIC",
+        "include_config": True,
+        "catch_exceptions": False,
+        "meta": None,
+        "row_condition": None,
+        "condition_parser": None,
+    }
+
+    domain_keys = (
+        "batch_id",
+        "row_condition",
+        "condition_parser",
+    )
+
+    def validate_configuration(
+        self, configuration: Optional[ExpectationConfiguration]
+    ) -> None:
+        super().validate_configuration(configuration)
+
+        query = configuration.kwargs.get("query") or self.default_kwarg_values.get(
```

Comment:
We need to add a type hint.  Thanks.

Reply:
🙇

Reply:
@austiezr I think he's referring to `query: str = configuration`...